### ISSUE TITLE
[Pattern] Centered Statement: Improvement

### ISSUE THREAD

**Issue by beafialho** [WordPress/twentytwentyfour#489]
I noticed this pattern wasn't looking like the designs, and the text also had an extra dash which shouldn't be there so I made the adjustments, the markup is below:

| Currently      | Designs |
| ----------- | ----------- |
|  <img width="1283" alt="Captura de ecrã 2023-09-22, às 11 53 00" src="https://github.com/WordPress/twentytwentyfour/assets/65220155/8aaed1e3-b285-403a-8d9d-76591222107b">    |    <img width="1285" alt="Captura de ecrã 2023-09-22, às 11 52 41" src="https://github.com/WordPress/twentytwentyfour/assets/65220155/bca8e3b9-d4e2-4a3d-ae12-6c5bb162ebb9">    |

```
<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|60","left":"var:preset|spacing|60","right":"var:preset|spacing|60"},"margin":{"top":"0","bottom":"0"}}},"backgroundColor":"base-2","layout":{"type":"constrained"}} -->
<div class="wp-block-group alignfull has-base-2-background-color has-background" style="margin-top:0;margin-bottom:0;padding-top:var(--wp--preset--spacing--60);padding-right:var(--wp--preset--spacing--60);padding-bottom:var(--wp--preset--spacing--60);padding-left:var(--wp--preset--spacing--60)"><!-- wp:group {"align":"wide","layout":{"type":"default"}} -->
<div class="wp-block-group alignwide"><!-- wp:spacer -->
<div style="height:100px" aria-hidden="true" class="wp-block-spacer"></div>
<!-- /wp:spacer -->

<!-- wp:paragraph {"align":"center","style":{"typography":{"lineHeight":"1.2","fontStyle":"normal","fontWeight":"400"}},"fontSize":"x-large","fontFamily":"heading"} -->
<p class="has-text-align-center has-heading-font-family has-x-large-font-size" style="font-style:normal;font-weight:400;line-height:1.2"><em>Études</em> is not confined to the past—we are passionate about the cutting edge designs shaping our world today.</p>
<!-- /wp:paragraph -->

<!-- wp:spacer -->
<div style="height:100px" aria-hidden="true" class="wp-block-spacer"></div>
<!-- /wp:spacer --></div>
<!-- /wp:group --></div>
<!-- /wp:group -->
```

**Comment by shail-mehta** [comment]
**PR:** https://github.com/WordPress/twentytwentyfour/pull/496

**Comment by beafialho** [comment]
This has been addressed in #496, I'm closing this issue
